emily/handler/src/database/entries/chainstate.rs

```diff
@@ -233,3 +240,88 @@ impl PrimaryIndexTrait for SpecialApiStateIndexInner {
         &settings.chainstate_table_name
     }
 }
+
+// Chainstate by bitcoin height --------------------------------------------------------------
+
+/// Chainstate table entry key. This is the secondary index key.
+#[derive(Clone, Default, Debug, Eq, PartialEq, Hash, Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct ChainstateByBitcoinHeightEntryKey {
+    /// Bitcoin height
+    pub bitcoin_height: u64,
+    /// Stacks height
+    #[serde(rename = "Height")]
+    pub stacks_height: u64,
```

Comment:
nit: just realized that elsewhere these are called `bitcoin_block_height` and `stacks_block_height` in other structs. We should stick to that and use the `rename` thingy to make sure that the names are correct when reading and writing to the database.

Reply:
> You mean, we want to use everywhere in the emily only `stacks_block_height` and `bitcoin_block_height` or `stacks_height` and `bitcoin_height` but not a mix?

Well, ideally yes. But for this PR, I mean the new structs that we are adding should call the fields referring to the stacks block height `stacks_block_height` and similarly for the bitcoin block height field.

> If yes I'd say I agree it would be a bit nicier, but we unfortunately don't follow this rule already ... So I think we can keep it as is

I don't quite follow. Doesn't this example show that we should use `stacks_block_height`? Also, assuming that this struct uses some other random name for the field, I don't think that fact supports the conclusion that we should be adding yet another name to refer to the same thing.